sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -42,11 +42,21 @@ class DatalabAppElement extends Polymer.Element {
    */
   public routeData: Object;
 
+  private _boundResizeHandler: EventListenerObject;
+
   constructor() {
     super();
 
     // Set the pattern once to be the current document pathname.
     this.rootPattern = (new URL(this.rootPath)).pathname;
+
+    this._boundResizeHandler = this._resizeHandler.bind(this);
+    window.addEventListener('resize', this._boundResizeHandler, true);
+
+    // Will be called after the custom element is done rendering.
+    window.addEventListener('WebComponentsReady', () => {
```

Comment:
Not sure what you mean. Line 54 only attaches the event listener, it doesn't call it. Line 58 calls the resize handler explicitly to set the initial size on element load.